index.Rmd

```diff
@@ -1,30 +1,44 @@
 --- 
 title: "Einführung in R"
 author: |
- Anna-Lena Schubert,
- Jan Goettmann,
  José Carlos García Alanis,
- Meike Steinhilber,
+ Jan Goettmann,
  Cordula Hunt,
- Florian Kobylka
+ Florian Kobylka,
+ Anna-Lena Schubert,
+ Meike Steinhilber,
+ 
 date: "`r Sys.Date()`"
 site: bookdown::bookdown_site
 documentclass: book
 bibliography: [book.bib, packages.bib]
 url: https://amd-lab.github.io/R-Kurs-Unterlagen/index.html
 # cover-image: path to the social sharing image like images/cover.jpg
 description: |
-  Dies sind R Kurs Unterlagen für Lehre in verschiedenen studentischen
-  Veranstaltungen.
+  Dies ist eine Einführung in R, die in verschiedenen Lehrveranstaltungen der Arbeitseinheiten *Analyse und Modellierung komplexer Daten* und *Methodenlehre und Statistik* des Psychologischen Instituts der Johannes Gutenberg-Universität Mainz genutzt werden.
 
 link-citations: yes
 github-repo:  AMD-Lab/R-Kurs-Unterlagen
 
 ---
 
-# Über dieses Buch
+# Über dieses Skript
+
+Dies ist eine Einführung in die Programmiersprache `R`, die in verschiedenen Lehrveranstaltungen der Arbeitseinheiten *Analyse und Modellierung komplexer Daten* und *Methodenlehre und Statistik* des Psychologischen Instituts der Johannes Gutenberg-Universität Mainz genutzt werden kann. `R` ist eine unter Psycholog:innen weit verbreitete Programmiersprache, mit deren Hilfe unterschiedlichste Datenarten aufbereitet und analysiert werden können. Das Programm ist *Open Source* und es gibt eine riesige Community von Entwickler:innen, die an Erweiterungen für die Software arbeiten, um Forschenden das (datenanalytische) Leben zu erleichtern.
+
+Dieses Skript wurde als Begleitmaterial für verschiedene Lehrveranstaltungen entwickelt, in denen mit `R` gearbeitet wird. Es kann als Kursmaterial, aber auch als Nachschlagewerk verwendet werden. Nicht alle Inhalte des Skripts sind für jeden Kurs relevant. Ihre Kursleitung wird Ihnen mitteilen, wann Sie welche Teile des Skripts bearbeiten sollen. Sie können sich natürlich darüber hinaus jederzeit weiter umschauen, welche Möglichkeiten `R` noch bietet!
+
+<center>
+
+![Ein digitaler Cartoon mit zwei Illustrationen: Die obere zeigt das R-Logo mit einem furchteinflößenden Gesicht und ein kleines, verängstigtes, wuscheliges Monster, das unter einer dunklen Gewitterwolke eine weiße Fahne hochhält, um sich zu ergeben. Der Text darüber lautet: "at first I was like…". Der untere Cartoon zeigt ein freundliches, lächelndes R-Logo, das aufspringt, um einem glücklichen, unscharfen Monster unter einer lächelnden Sonne und neben bunten Blumen ein High-Five zu geben. Der Text über der unteren Illustration lautet "but now it’s like…"](_static/r_first_then.png){width="400"}[^source]
+
```

Comment:
and here too, or am I missing something? it's also the same text as in `Einführung.Rmd`. It's just the name for the reference? perhaps we could just name them, ![allison1] and ![allison2]

then it would look something like this:
...
<center>
![allison1](_static/r_first_then.png){width="400"}[^allison1]
</center>
[^allison1]: Artwork by [Allison Horst](https://github.com/allisonhorst/stats-illustrations)
...